test/lit/passes/O1_skip.wast

```diff
@@ -1,6 +1,3 @@
-;; NOTE: Assertions have been generated by update_lit_checks.py --all-items and should not be edited.
-;; NOTE: This test was ported using port_passes_tests_to_lit.py and could be cleaned up.
```

Comment:
Is this a mistake?

Reply:
No, it's intentional - this test now has a manual `CHECK-NOT` so it can't be auto-updated (the update removes that line!). I missed this when landing and it's annoying in local work. I can open a separate PR, but it is trivial?

Reply:
Moved out.